src/request.cc

```diff
@@ -19,11 +19,6 @@ std::string Request::extension_to_type(std::string ext) {
   return "text/plain";
 }
 
-std::string Request::update_uri(std::string new_uri) {
```

Comment:
I added this before because I thought it would be useful but didn't end up needing it